libraries/AP_HAL_ChibiOS/hwdef/ORBITH743/hwdef.dat

```diff
@@ -0,0 +1,217 @@
+PE9  IMU2_CS CS
+PE12 SPI4_SCK SPI4
+PE13 SPI4_MISO SPI4
+PE14 SPI4_MOSI SPI4
+
+# one I2C bus
+I2C_ORDER I2C1
+
+# I2C1 For BARO
+PB8 I2C1_SCL I2C1
+PB9 I2C1_SDA I2C1
+
+# ADC for Power
+PC0 BATT_VOLTAGE_SENS ADC1 SCALE(1)
+PC1 BATT_CURRENT_SENS ADC1 SCALE(1)
+PC4 BATT2_VOLTAGE_SENS ADC1 SCALE(1)
+PA4 BATT2_CURRENT_SENS ADC1 SCALE(1)
+
+define HAL_BATT_MONITOR_DEFAULT 4
+define HAL_BATT_VOLT_PIN 10
+define HAL_BATT_CURR_PIN 11
+
+define HAL_BATT2_VOLT_PIN 4
+define HAL_BATT2_CURR_PIN 18
+
+define HAL_BATT_VOLT_SCALE 10.1
+define HAL_BATT_CURR_SCALE 80
+define HAL_BATT2_VOLT_SCALE 10.1
+define HAL_BATT2_CURR_SCALE 80
+
+# ADC for rssi
+PC5 RSSI_ADC ADC1
+define BOARD_RSSI_ANA_PIN 8
+
+# Define LEDs
+# blue LED0 marked as ACT
+# red LED1 marked as B/E
+PE3 LED0 OUTPUT LOW GPIO(90) # blue
+PE4 LED1 OUTPUT LOW GPIO(91) # red
+define AP_NOTIFY_GPIO_LED_2_ENABLED 1
+define HAL_GPIO_A_LED_PIN 90
+define HAL_GPIO_B_LED_PIN 91
+
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 USART1 USART2 USART3 UART4 UART5 USART6 UART7 UART8
+
+# USART1 (ESC Telemetry), SERIAL1
+PA10  USART1_RX USART1 NODMA
+PB14  USART1_TX USART1 NODMA
+define DEFAULT_SERIAL1_PROTOCOL SerialProtocol_ESCTelemetry
+
+# USART2 (Telemetry), SERIAL2
+PD6 USART2_RX USART2
+PD5 USART2_TX USART2
+define DEFAULT_SERIAL2_PROTOCOL SerialProtocol_None
+
+# USART3 (VTX), SERIAL3
+PD9 USART3_RX USART3
+PD8 USART3_TX USART3
+define DEFAULT_SERIAL3_PROTOCOL SerialProtocol_MSP_DisplayPort
+
+# UART4 (Telemetry), SERIAL4
+PD0 UART4_RX UART4
+PD1 UART4_TX UART4
+define DEFAULT_SERIAL4_PROTOCOL SerialProtocol_None
+
+# UART5 (RC input), SERIAL5
+PB12  UART5_RX UART5
+PB13  UART5_TX UART5
+define DEFAULT_SERIAL5_PROTOCOL SerialProtocol_RCIN
+
+# USART6 (GPS), SERIAL6
+PC7 USART6_RX USART6
+PC6 USART6_TX USART6
+define DEFAULT_SERIAL6_PROTOCOL SerialProtocol_GPS
+
+# UART7 (Telemetry), SERIAL7
+PE7 UART7_RX UART7
+PE8 UART7_TX UART7
+define DEFAULT_SERIAL7_PROTOCOL SerialProtocol_None
+
+# UART8 (Telemetry), SERIAL8
+PE0 UART8_RX UART8
+PE1 UART8_TX UART8
+define DEFAULT_SERIAL8_PROTOCOL SerialProtocol_None
+
+
+# Motors
+PA0  TIM5_CH1 TIM5  PWM(1)  GPIO(50) BIDIR   //M1                  
+PA1  TIM5_CH2 TIM5  PWM(2)  GPIO(51)         //M2
+PA2  TIM5_CH3 TIM5  PWM(3)  GPIO(52) BIDIR   //M3
+PA3  TIM5_CH4 TIM5  PWM(4)  GPIO(53)         //M4
+PB0  TIM3_CH3 TIM3  PWM(5)  GPIO(54) BIDIR   //M5
+PB1  TIM3_CH4 TIM3  PWM(6)  GPIO(55)         //M6
+PB4  TIM3_CH1 TIM3  PWM(7)  GPIO(56) BIDIR   //M7
+PB5  TIM3_CH2 TIM3  PWM(8)  GPIO(57)         //M8
+
+# Servos
+PA15  TIM2_CH1 TIM2  PWM(9)  GPIO(58) NODMA  //S1
+PB3   TIM2_CH2 TIM2  PWM(10) GPIO(59) NODMA  //S2
+PD12  TIM4_CH1 TIM4  PWM(11) GPIO(60) NODMA  //S2
+PD13  TIM4_CH2 TIM4  PWM(12) GPIO(61) NODMA  //S2
+
+# LED strip
+PA8  TIM1_CH1 TIM1  PWM(13) GPIO(62) NODMA
+
+# PWM output for buzzer/Beeper  
+PB7 BUZZER OUTPUT LOW PULLDOWN GPIO(80)
+define HAL_BUZZER_PIN 80
+
+# GPIOs for 9V DCDC control (HIGH:on; LOW:off)
+PE2  PINIO1 OUTPUT GPIO(81) HIGH
```

Comment:
Suggest you put this on a relay so that people can assign an RC switch easily

Reply:
Done in #30237